grails-app/views/stockMovement/show.gsp

```diff
@@ -1,4 +1,6 @@
-<%@ page import="org.pih.warehouse.core.Role; org.pih.warehouse.auth.AuthService" %>
+<%@ page import="org.pih.warehouse.requisition.RequisitionStatus" %>
+<%@ page import="org.pih.warehouse.core.Role" %>
+<%@ page import="org.pih.warehouse.auth.AuthService" %>
```

Comment:
Remove this and pass the `session.user.id` to the convenience method you added to the OutboundStockMovement and StockMovement